src/main/java/com/google/devtools/build/lib/analysis/RunfilesSupport.java

```diff
@@ -132,15 +135,19 @@ private static RunfilesSupport create(
       runfilesInputManifest = null;
       runfilesManifest = null;
     }
+    Artifact repoMappingManifest =
+        createRepoMappingManifestAction(ruleContext, runfiles, owningExecutable);
```

Comment:
Just noticed that we aren't adding the repo mapping manifest to the runfiles manifest, which makes it more difficult to find it at runtime on Windows. If we move this call further up, we could pass the Artifact into `SourceManifestAction`.